Marketing.md

```diff
@@ -0,0 +1,36 @@
+# Marketing
+
+## Product Marketer Manager (Developer doing Marketing)
+
+> I am the voice of the customer, and I am the glue between engineering and marketing to drive interest and people trying our product
+
+**I have a deep knowledge of the ethos, patterns and principles of Octopus Deploy and I understand its benefits for teams and the problems it solves**
+
+* I am the authority on our customers, who are they, how they buy and why they pick Octopus (buying criteria).
```

Comment:
Consider: I am an authority on our customers being a developer myself, and understand what motivates them to choose Octopus